ts/features/bonus/cgn/screens/merchants/CgnMerchantCategoriesListScreen.tsx

```diff
@@ -63,11 +63,10 @@ export const CgnMerchantCategoriesListScreen = () => {
   );
 
```

Comment:
Then here we could have an effect that sets the `isRefreshing` state to `false`, something like this:

```typescript
  React.useEffect(() => {
    if (pot.isSome(potCategories) && !pot.isLoading(potCategories)) {
      setIsRefreshing(false);
    }
  }, [potCategories]);
```